frontend/src/pages/ApplicationsPage.tsx

```diff
@@ -64,8 +64,8 @@ const ApplicationsPage: React.FC<ApplicationsPageProps> = ({
               <Text component="h1" data-testid="app-page-title">
                 {title}
               </Text>
-              {jobReferenceName}
-              {description && <Text component="p">{description}</Text>}
+              <div>{subtext}</div>
+              <div>{description}</div>
```

Comment:
Didn't test this rendering, just fixed some naming -- need to test this.

TODO